integration/action-test.ts

```diff
@@ -254,56 +262,54 @@ describe("actions", () => {
   // TODO: figure out what the heck is wrong with this test...
   // For some reason the error message is "Unexpect Server Error" in the test
   // but if you try the app in the browser it works as expected.
-  it.skip("rejects too big of an upload with JavaScript", async () => {
+  test.skip("rejects too big of an upload with JavaScript", async ({
+    page,
+  }) => {
+    let app = new PlaywrightFixture(appFixture, page);
     await app.goto(`/${HAS_FILE_ACTIONS}`);
 
     let html = await app.getHtml("#action-text");
     expect(html).toMatch(WAITING_VALUE);
 
-    let fileInput = await app.page.$("#file");
-    await fileInput!.uploadFile(
+    await app.uploadFile(
+      "#file",
       path.resolve(__dirname, "assets/touploadtoobig.txt")
     );
 
-    await app.page.click("button[type=submit]");
-    await app.page.waitForSelector("#actions-error-boundary");
+    await page.click("button[type=submit]");
+    await page.waitForSelector("#actions-error-boundary");
 
     let text = await app.getHtml("#actions-error-text");
     expect(text).toMatch(
       `Field "file" exceeded upload size of ${MAX_FILE_UPLOAD_SIZE} bytes`
     );
 
-    expect(consoleError).toHaveBeenCalledTimes(1);
-    let errorObject = expect.objectContaining({
-      message: expect.stringMatching(/exceeded upload size/i),
+    let logs: string[] = [];
+    page.on("console", (msg) => {
+      logs.push(msg.text());
     });
-    expect(consoleError).toHaveBeenCalledWith(errorObject);
-    consoleError.mockClear();
+    expect(logs).toHaveLength(1);
+    expect(logs[0]).toMatch(/exceeded upload size/i);
   });
 
-  describe("without JavaScript", () => {
-    let restore: Awaited<ReturnType<typeof app.disableJavaScript>>;
-    beforeEach(async () => {
-      restore = await app.disableJavaScript();
-    });
-    afterEach(async () => {
-      await restore();
-    });
+  test.describe("without JavaScript", () => {
+    test.use({ javaScriptEnabled: false });
```

Comment:
I *think* this should this be in a `beforeAll` hook.

Reply:
you'd think, but no 😉 https://playwright.dev/docs/test-configuration#local-configuration